blocks/carousel/carousel.css

```diff
@@ -270,4 +381,62 @@ main .section div.carousel-wrapper {
         text-align: left;
         margin: 0;
     }
+
+    .carousel.case-study .carousel-nav {
+        opacity: 1;
+        top: 36%;
+        background: unset;
+    }
+    
+    .carousel.case-study .carousel-nav.carousel-nav-prev {
+        transform: unset;
+        background: url('./../../icons/carousel-arrow-left.svg') no-repeat;
+        background-size: contain;
+        width: 101px;
```

Comment:
I know that this `101px` value came from the design specs, but it feels odd and wrong to me. I think we should make this even value of `100px`. @sachinmesh Can you help update this?

![Screenshot 2023-08-30 at 12 15 41 PM](https://github.com/hlxsites/merative2/assets/1815714/5d3d7805-eb6b-48dc-8d10-1075de1f7ee9)

cc: @aavilamer

Reply:
Made changes in Design system with 100 x 100 px 
<img width="282" alt="image" src="https://github.com/hlxsites/merative2/assets/124026684/57bc1c6f-9287-4aef-81d2-f72caca78187">

@proeung @nimithshetty17